config/config.example.toml

```diff
@@ -267,6 +267,7 @@ stripe.base_url_file_upload = "https://files.stripe.com/"
 trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 trustpay.base_url_bank_redirects = "https://aapi.trustpay.eu/"
 tsys.base_url = "https://stagegw.transnox.com/"
+unifiedauthenticationservice.base_url = "https://uas.internal.staging.mum.juspay.net"
```

Comment:
Can we have this as snake case please? Becomes difficult to read when the names get longer.